apps/src/codebridge/utils/getOpenFiles.ts

```diff
@@ -0,0 +1,20 @@
+import {ProjectType} from '@codebridge/types';
+
+import {shouldShowFile} from './fileUtils';
+import {sortFilesByName} from './sortFilesByName';
+
+export const getOpenFiles = (project: ProjectType) => {
+  if (project.openFiles) {
```

Comment:
can you file a follow-up task to use `openFiles` as the single source of truth?

Reply:
Filed as https://codedotorg.atlassian.net/browse/CT-699